GlobalStyles/GlobalStyles.js

```diff
@@ -5,6 +5,9 @@ const GlobalStyles = createGlobalStyle`
     body {
         padding: 0;
         margin: 0;
+        position: relative;
+        min-height: 100%; 
```

Comment:
from our mobile layout workshop with Julia I noted `height: 100vh;` not sure if this works as well?!

Reply:
I tried it, but it's changing my browser view in a weird way, but % is doing it for me.